tools/verdict/com.ge.research.osate.verdict/src/com/ge/research/osate/verdict/handlers/CRVHandler.java

```diff
@@ -88,16 +86,25 @@ public void run() {
 		return null;
 	}
 
-	public static boolean runBundle(String bundleJar, String dockerImage, String inputPath, String outputPath,
-			String aadl2imlbin, String kind2bin, String propertySet) throws IOException {
+	/**
+	 * Calls Aadl2Vdm translator and writes model to vdmFile
+	 * @param dir
+	 * @param vdmFile
+	 */
+	public static void runAadl2Vdm(File dir, File vdmFile) {
+		Aadl2Vdm aadl2vdm = new Aadl2Vdm();
+		Model model = aadl2vdm.execute(dir);
```

Comment:
Need call to agree2vdm.execute here. Agree2vdm.execute first populates the vdm with aadl data by invoking aadl2vdm and then populates the AGREE data. However, Agree2vdm.execute has to be cleaned up to remove test code.